src/users/components/invite-user-permissions/invite-user-permission.component.html

```diff
@@ -1,36 +1,36 @@
   <!--permissions -->
   <ng-container [formGroup]="inviteUserForm">
     <lib-gov-uk-form-group-wrapper
-      [config]="{hint: 'Choose at least one action the user can do. You can change this later.', legend: 'Permissions', key: 'roles'}"
+      [config]="{hint: 'Choose at least one action the user can do. You can change this later.', legend: 'What user Permissions would you like to approve?', key: 'roles'}"
       [formGroupName]="'roles'"
       [error]="errorMessages">
 
       <div class="govuk-checkboxes">
         <lib-gov-checkbox
           [group]="inviteUserForm.controls['roles']"
           [isChecked]="isPuiCaseManager"
-          [config]="{value: 'pui-case-manager',  label: 'Manage cases', name: 'roles', hint: 'View, create and progress cases.', focusOn: 'roles'}">
+          [config]="{value: 'pui-case-manager',  label: 'Manage cases', name: 'roles', hint: 'View, create and progress cases.', focusOn: 'roles', spanText: 'Permit users to '}">
         </lib-gov-checkbox>
 
         <lib-gov-checkbox
           [group]="inviteUserForm.controls['roles']"
           [isChecked]="isPuiUserManager"
           [config]="{value: 'pui-user-manager', label: 'Manage users', name: 'roles',
-          hint: 'Invite users.'}">
+          hint: 'Invite users.', spanText: 'Permit users to '}">
         </lib-gov-checkbox>
 
         <lib-gov-checkbox
           [group]="inviteUserForm.controls['roles']"
           [isChecked]="isPuiOrganisationManager"
           [config]="{value: 'pui-organisation-manager', label: 'Manage organisation', name: 'roles',
-          hint: 'View the organisation name and address.'}">
+          hint: 'View the organisation name and address.', spanText: 'Permit users to '}">
         </lib-gov-checkbox>
 
         <lib-gov-checkbox id="checkbox-grant-case-access-admin" *ngIf="grantCaseAccessAdmin$ | async"
           [group]="inviteUserForm.controls['roles']"
           [isChecked]="isCaseAccessAdmin"
           [config]="{value: 'pui-caa', label: 'Case access administrator', name: 'roles',
-          hint: 'Manage case access for all users within your organisation. You can assign this role to any user in your organisation. However, your service might not currently support this role. You will receive further guidance once it becomes available.'}">
+          hint: 'Manage case access for all users within your organisation. You can assign this role to any user in your organisation. However, your service might not currently support this role. You will receive further guidance once it becomes available.', spanText: 'Permit users to '}">
```

Comment:
Not sure if lint will pick this one up but it's a _really_ long line and not easy to read as it requires sideways scrolling...

Reply:
split the line